spec/features/shelterreviews/delete_a_shelter_review_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe "when I visit a shelter show page ther is a link to delete a review" do
+  describe "when I click the link i am returned to the shelters show page" do
+    it "and the review is no longer there" do
+
+      shelter1 = Shelter.create(name: "Dog-Haven",
+                               address:  "1234 Barkers Way",
+                               city: "Beagle",
+                               state: "MA",
+                               zip: "01001")
+
+      review1 = shelter1.reviews.create!(review_image: "https://via.placeholder.com/150",
+                                         title: "test review 1",
+                                         rating: 1,
+                                         content: "this review is average at best.")
```

Comment:
can we add a second review to this test and verify afterward that the deleted review is gone but the other still remains?